lib/kitchen/driver/dokken.rb

```diff
@@ -374,8 +375,8 @@ def start_data_container(state)
           # locally built image, must use short-name
           "Image" => short_image_path(data_image),
           "HostConfig" => {
-            "PortBindings" => port_bindings,
-            "PublishAllPorts" => true,
+            "PortBindings" => data_port_bindings,
+            "PublishAllPorts" => config[:data_ssh_port].nil?,
```

Comment:
change to `self[:data_ssh_port].nil?`

Reply:
form the surrounding code config[:option] should work. 

Bit confused why it isn't working here! Any ideas?